django/santropolFeast/member/models.py

```diff
@@ -159,12 +159,25 @@ class Client(models.Model):
         (DECEASED, _('deceased')),
     )
 
+    FACTURATION_TYPE = (
+        ('default', _('default')),
+        ('low income', _('low income')),
```

Comment:
Be carefull we do not write in English but we just create id which will then be translated.
Here you need to have ```low_income``` per example